apps/clearance_ui/src/components/clearance_inspector/ConclusionFormWrapper.tsx

```diff
@@ -59,7 +59,7 @@ const ConclusionFormWrapper = ({
                         <Label className="clearance-label">
```

Comment:
Not directly related to this PR, but what's the reason for this being a class instead of using Tailwind, and where does the class come from? A code comment for that could be good.

Reply:
That's @lamppu 's work, so could you comment on your decision of making this a separate class in `globals.css`; it it just for avoiding code repetition?

Reply:
In `globals.css`:
```
.clearance-label {
    @apply rounded-sm p-0.5 font-bold text-[#313C9B] dark:text-[#C6CAED] !important;
}
```

Reply:
Makes sense. IMO more Tailwindy way would be to create a component (e.g., `<ClearanceLabel>`) that uses the tailwind classes, and then reuse that. That way it's easier to see how it's styled, and helps prevent orphan CSS classes, if that is some day removed.